### PR TITLE
Release 2.9.6-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming second beta of the v2.9.6 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
--- Feature Flags have not changed - check runs still beta.
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
Metrics added for check runs - PR are merged/deployed.
- [central](https://github.com/github/central/pull/565)
- [desktop.github.com](https://github.com/github/desktop.github.com/pull/164)

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.9.6-beta2": [
+      "[Fixed] Stashing dialog no longer hangs when intiating cherry-pick - #13419",
+      "[Fixed] Rebase no longer hangs after conflicts resolved when intiated through pull conflict error - #13204",
+      "[Fixed] Able to discard files and repositories - #13433",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Discarding files and repositories would fail to move items to trash - #13433",
```

**Review by niik — CHANGES_REQUESTED** [review]
nit-⛏️

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.9.6-beta2": [
+      "[Fixed] Stashing dialog no longer hangs when intiating cherry-pick - #13419",
+      "[Fixed] Rebase no longer hangs after conflicts resolved when intiated through pull conflict error - #13204",
+      "[Fixed] Discarding files and repositories would fail to move items to trash - #13433",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Discarding files and repositories successfully moves items to trash - #13433",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+      "[Improved] Added check runs overall completeness indicator in check run popover header - #13423"
```
Comment (billygriffin) [inline changelog.json:7]:
```suggestion
      "[Improved] Added check runs overall progress indicator in check run popover header - #13423"
```

**Review by billygriffin — COMMENTED** [review]
Thanks! Looks good overall, a couple minor suggestions.

File: changelog.json (hunk 3 above)
Comment (tidy-dev) [reply changelog.json:7]: Discussed in slack - not exactly a progress indicator since it is a pie chart of all statuses.

**Review by billygriffin — DISMISSED** [review]
🚀

**Review by niik — DISMISSED** [review]

**Review by niik — APPROVED** [review]

**Pull request merged**